packages/discord.js/src/client/actions/MessagePollVoteAdd.js

```diff
@@ -11,11 +14,33 @@ class MessagePollVoteAddAction extends Action {
     const message = this.getMessage(data, channel);
     if (!message) return false;
 
-    const { poll } = message;
+    const includePollPartial = this.client.options.partials.includes(Partials.Poll);
+    if (message.partial && !includePollPartial) return false;
+
+    if (!message.poll && includePollPartial) {
+      message.poll = new Poll(
+        this.client,
+        { ...data, question: { text: '' }, answers: [], partial: true },
+        message,
+        channel,
+      );
+    }
+
+    const includePollAnswerPartial = this.client.options.partials.includes(Partials.PollAnswer);
+    if (message.partial && includePollPartial && !includePollAnswerPartial) return false;
+
+    let answer = message.poll?.answers?.get(data.answer_id);
+    if (!answer && message.poll) {
+      const pollAnswer = new PollAnswer(this.client, data, message.poll);
```

Comment:
I'll combine these two poll partials `if` statements so there isn't any unnecessary extra work when either partial is disabled.